subset/switches/src/main/java/switchtest/SwitchInterrogator.java

```diff
@@ -445,6 +439,49 @@ private String cleanShowRunData(String data) {
     return data;
   }
 
+  private void validateTests() {
+    try {
+      String link_status = show_interface_data[1];
+      String dropped_packets = show_interface_data[12];
```

Comment:
this is ok for now, but generally speaking you want to keep random constants out of the code blocks, so it should just be

dropped_packets = show_interface_data[DROPPED_PACKETS_INDEX],

in which case then you don't need the intermediate variable.